net-irc/inspircd/inspircd-3.13.0.ebuild

```diff
@@ -0,0 +1,113 @@
+# Copyright 1999-2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit systemd toolchain-funcs
+
+DESCRIPTION="Inspire IRCd - The Stable, High-Performance Modular IRCd"
+HOMEPAGE="https://www.inspircd.org/"
+SRC_URI="https://github.com/inspircd/inspircd/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+SRC_URI+=" https://github.com/clinew/gentoo-distfiles/raw/master/inspircd-${PV}-fix-path-builds.patch.bz2 https://github.com/clinew/gentoo-distfiles/raw/master/inspircd-${PV}-gentooize-logrotate.patch.bz2"
```

Comment:
just style but this line sure is long, didn't even see the 2nd patch -- may want a 2nd += or do it like:
```bash
SRC_URI="
	https://github.com/inspircd/inspircd/archive/v${PV}.tar.gz -> ${P}.tar.gz
	https://github.com/clinew/gentoo-distfiles/raw/master/inspircd-${PV}-fix-path-builds.patch.bz2
	https://github.com/clinew/gentoo-distfiles/raw/master/inspircd-${PV}-gentooize-logrotate.patch.bz2"
```

Reply:
Agreed; I'll fix it up.